spec/amp-var-substitutions.md

```diff
@@ -76,6 +76,28 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
 
+### SOURCE_URL
+
+Use the special string `SOURCE_URL` to add the source URL of the current document to the URL
+
+For instance:
+```html
+<amp-pixel src="https://foo.com/pixel?href=SOURCE_URL"></amp-pixel>
+```
+may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
+
+The source URL is extracted from the proxy URL if the document is proxied or otherwise the original document is returned.
```

Comment:
I would move this up above the example and perhaps expand this with a clearer example. You could say something like:

The source URL is extracted from the proxy URL if the document is being served from a proxy. Otherwise the original document URL is returned. For instance, if the URL is served via the proxy https://cdn.ampproject.org from the URL https://cdn.ampproject.org/c/s/example.com/page.html, then SOURCE_URL would return https://example.com/page.html.